modules/release-notes/ref-release-notes-deprecated-functionalities.adoc

```diff
@@ -7,7 +7,7 @@ This section lists deprecated functionalities in {product} {product-version}.
 [id="deprecated-functionality-rhidp-3545"]
 == `./dynamic-plugins/dist/janus-idp-backstage-plugin-aap-backend-dynamic` plugin is deprecated
 
-The `./dynamic-plugins/dist/janus-idp-backstage-plugin-aap-backend-dynamic` plugin has been deprecated and will be removed in the next release. You can link:https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.5/html-single/using_ansible_plug-ins_for_red_hat_developer_hub/index[use Ansible plug-ins for {product-very-short}] instead.
+The `./dynamic-plugins/dist/janus-idp-backstage-plugin-aap-backend-dynamic` plugin has been deprecated and will be removed in the next release. You can link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/{product-version}/html-single/using_dynamic_plugins/index#using-ansible-plug-ins-for-red-hat-developer-hub[use Ansible plug-ins for {product}] instead.
```

Comment:
this change makes people read a warning and then have to click through to https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.4/html/using_ansible_plug-ins_for_red_hat_developer_hub/index which is the old version.

Why force them to click twice when we can save them time and effort? 

-1 for this UX degradation.

Reply:
sure but the actual content for how to configure the plugin is at https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.5/html-single/using_ansible_plug-ins_for_red_hat_developer_hub/index because this plugin is owned by the RH Ansible team. So why make people click twice?